### ISSUE TITLE
Errors importing class Cleaners from paso.pre.cleaners

### ISSUE THREAD

**Issue by DarioSpiller** [bcottman/paso#17]
Hi Bruce,

I'm a PostDoc at the European Space Agency. I'm trying to use your library as it seems very interesting and useful. However, when trying your example lessons, in particular Part 1 - Uncommon Data Cleaners for your Real-World Machine Learning Projects, I got the following errors. 

`from paso.pre.cleaners import Cleaners

c = Cleaners()

dataset['asv'] = 999.0

c.values_to_nan(dataset,values=[999.,11999.],inplace=True)

dataset.head()`

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
~/Scrivania/paso-master in <module>
      2 c = Cleaners()
      3 dataset['asv'] = 999.0
----> 4 c.values_to_nan(dataset,values=[999.,11999.],inplace=True)
      5 dataset.head()

AttributeError: 'Cleaners' object has no attribute 'values_to_nan'

It seems python does not recognizes the methods associated to the class Cleaners. Can you help me? I'm using python 3.7.3 on Ubuntu 18.04.4 LTS

**Comment by bcottman** [comment]
Yes, I will look at it. I will be porting all this to Photonai. I recommend you look at that package.

-Bruce
